KerbalVR_Mod/KerbalVR/InteractionSystem/KerbalVR_Hand.cs

```diff
@@ -372,6 +412,37 @@ protected void Update()
 
 			}
 		}
+
+		private void SyncCollider()
+		{
+			fingertipTransform.position = indexTargetTransform.TransformPoint(CurrentProfile.fingertipOffset);
+			thumbTransform.position = thumbTargetTransform.position;
+			palmTransform.position = palmTargetTransform.TransformPoint(CurrentProfile.gripOffset);
+			
+			fingertipTransform.rotation = indexTargetTransform.rotation;
+			palmTransform.rotation = handObject.transform.rotation;
```

Comment:
we wouldn't need to do any of this (other than pinch) if the colliders were attached

Reply:
Fixed by [c35e6bc](https://github.com/JonnyOThan/Kerbal-VR/pull/172/commits/c35e6bc15153e8ec6391f1a7d6308ce51420088a)